testcafe/tests/pages/drive-viewer/drive-viewer-model.js

```diff
@@ -92,7 +77,7 @@ export default class ViewerDrive extends Viewer {
   }
 
   //download using the common download button
-  async openFileAndCheckCommonViewerDownload(fileName) {
+  async openFileAndcheckCommonViewerDownload(fileName) {
```

Comment:
Why was this function renamed?

Reply:
Oh, this looks like a rebase/merge error!